src/tsolvers/lasolver/LASolver.cpp

```diff
@@ -383,7 +382,7 @@ bool LASolver::assertLit( PtAsgn asgn, bool reason )
         return getStatus();
     }
     if (deduced[t.getVar()] != l_Undef && deduced[t.getVar()].deducedBy == id) {
-        is_reason = true; // This is a conflict!
+        // MB: what should happen here?
```

Comment:
Not sure...  Is this some case that has become obsolete or should the solver be trying to construct a conflict clause?

Reply:
I don't have a good understanding of the purpose and usage of "deduced" field.
I don't think this case happens in current code.